docs/src/docs/asciidoc/user/clustered-cache.adoc

```diff
@@ -143,6 +143,11 @@ In this example, a dedicated pool of 32MB is allocated for clustered-cache from
 <7> Configures another cache (`shared-cache-2`) that shares the resource pool (`resource-pool-a`) with `shared-cache-1`.
 <8> Creates fully initialized cache manager with the clustered caches.
 
+NOTE:  When multiple caches are sharing a shared pool, eviction operation called by a cache evicts elements of that cache
```

Comment:
evictions

Reply:
nothing on eviction now in the new version